WebApiGateway/WebApiGateway.Api/Clients/DecisionClient.cs

```diff
@@ -30,7 +31,13 @@ public async Task<RegulatorDecision> GetDecisionAsync(string queryString)
         {
             await ConfigureHttpClientAsync();
 
-            var response = await _httpClient.GetAsync($"decisions{queryString}");
+            var uriBuilder = new UriBuilder(_httpClient.BaseAddress)
```

Comment:
@Mazar-Shaik Please revert this change and ignore the SQ error

Reply:
Added uriBuilder.Uri.LocalPath instead of uriBuilder.Path

Reply:
please add a unit test